docs/Configuring-Pantheon/Proof-of-Authority.md

```diff
@@ -0,0 +1,80 @@
+description: Pantheon Clique Proof-of-Authority (PoA) consensus protocol implementation
+path: blob/master/ethereum/core/src/main/resources/
+source: rinkeby.json
+<!--- END of page meta data -->
+
+# Proof of Authority
+
+Pantheon implements the Clique Proof-of-Authority (PoA) consensus protocol. Clique is used by the Rinkeby testnet and can be used for private networks. 
+
+In PoA networks, transactions and blocks are validated by approved accounts, known as signers. Signers take turns to create the next block. Existing signers propose and vote to add or remove signers. 
+
+## Genesis File
+
+To use Clique in a private network, Pantheon requires a PoA genesis file. When connecting to Rinkeby, Pantheon uses the [`rinkeby.json`](https://github.com/PegaSysEng/pantheon/blob/master/ethereum/core/src/main/resources/rinkeby.json) genesis file in the `/pantheon/ethereum/core/src/main/resources` directory.
+
+A PoA genesis file defines properties specific to Clique:
+
+!!! example "Genesis JSON file (stripped)"
```

Comment:
Is stripped commonly used in this way?  It made me look twice - I assume we're using to to indicate the genesis file is incomplete?

Reply:
yes, well perhaps it's not the right word 🤣